tests/cql/CqlQueryTests.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://hl7.org/fhirpath/tests" xsi:schemaLocation="http://hl7.org/fhirpath/tests ../../testSchema/testSchema.xsd"
+	name="CqlIntervalOperatorsTest" reference="https://cql.hl7.org/09-b-cqlreference.html#interval-operators-3">
+    <group name="SimpleQueries">
+        <test name="NonListSource">
+            <expression>(4) l</expression>
+            <output>4</output>
+        <test>
```

Comment:
Missing /.  should be </test>.  Several places throughout file.

Reply:
Fixed.